app/services/order_tax_adjustments_fetcher.rb

```diff
@@ -32,7 +32,7 @@ def order_adjustments
   end
 
   def line_item_adjustments
-    table[:adjustable_id].eq(order.line_item_ids.join(','))
+    table[:adjustable_id].eq_any(order.line_item_ids)
```

Comment:
FYI I found out in https://rollout.io/blog/creating-advanced-active-record-db-queries-arel/ that `eq_any` changed in Rails 4.2. Apparently it didn't accept an empty collection anymore. However, in this case, there should always be at least a line item.